### ISSUE TITLE
pkgin avail | grep beats shows nothing

### ISSUE THREAD

**Issue by TyrfingMjolnir** [TritonDataCenter/pkgsrc-legacy#436]
However there is some work in progress here: https://wip.pkgsrc.org/cgi-bin/gitweb.cgi?p=pkgsrc-wip.git;a=tree;f=beats

**Comment by mamash** [comment]
wip packages are not automatically included in production builds and this one was a "will it build" one time experiment only.

**Comment by mamash** [comment]
I just updated the wip package to 5.x and briefly tested metricbeat and filebeat and they seem to work fine. So I think I can bring the bunch into our production packages soon, it just needs a lot of love in terms of making sure the software doesn't log and store to random directories, because it's rather stupid in that regard.

**Comment by TyrfingMjolnir** [comment]
Filebeat and Packetbeat is supposed to work according to this matrix https://www.elastic.co/support/matrix Does Metricbeat work as well?

**Comment by mamash** [comment]
I did some testing in December and metricbeats seems to work in general on SmartOS, but none of the built-in metrics (for cpu etc.) are supported on SunOS/SmartOS.

I updated the package to the latest 5.1.1 version, cleaned up the pathing and included a rudimentary SMF manifest. I also imported the package into the main pkgsrc tree, so >=2017Q1 releases will have it automatically.  I'll enable the package in the upcoming 2016Q4 package set explicitly.
